Disables dri for embedded platform.

Signed-off-by: Damian Wrobel <dwrobel@ertelnet.rybnik.pl>
---

Index: chromium-76.0.3809.132/build/config/linux/dri/BUILD.gn
===================================================================
--- chromium-76.0.3809.132.orig/build/config/linux/dri/BUILD.gn
+++ chromium-76.0.3809.132/build/config/linux/dri/BUILD.gn
@@ -5,14 +5,3 @@
 import("//build/config/linux/pkg_config.gni")
 
 assert(is_linux, "This file should only be referenced on Linux")
-
-pkg_config("dri") {
-  packages = [ "dri" ]
-  dri_driver_dir = exec_script(pkg_config_script,
-                               pkg_config_args + [
-                                     "--dridriverdir",
-                                     "dri",
-                                   ],
-                               "string")
-  defines = [ "DRI_DRIVER_DIR=\"$dri_driver_dir\"" ]
-}
