Index: chromium-83.0.4103.61/tools/v8_context_snapshot/v8_context_snapshot.gni
===================================================================
--- chromium-83.0.4103.61.orig/tools/v8_context_snapshot/v8_context_snapshot.gni
+++ chromium-83.0.4103.61/tools/v8_context_snapshot/v8_context_snapshot.gni
@@ -14,10 +14,10 @@ import("//v8/gni/v8.gni")
 declare_args() {
   # TODO(crbug.com/764576): Enable the feature on more environments.
   # Disable in win/cross builds since buliding Blink twice is slow.
-  use_v8_context_snapshot =
-      !is_chromeos && !is_android && !is_chromecast && !is_fuchsia &&
-      !(host_os == "mac" && current_cpu == "x86") &&
-      (v8_target_cpu == target_cpu || is_msan) && !(is_win && host_os != "win")
+  use_v8_context_snapshot = false
+  #    !is_chromeos && !is_android && !is_chromecast && !is_fuchsia &&
+  #    !(host_os == "mac" && current_cpu == "x86") &&
+  #    (v8_target_cpu == target_cpu || is_msan) && !(is_win && host_os != "win")
 }
 
 # We cannot use V8 context snapshot, if V8 doesn't use snapshot files.
